While executing file `Games/Thumgeon_II/main.py` an error occurred (<module> refers to the game file):

Traceback (most recent call last):
  File "/home/devy/Desktop/micropython-rp2350-1.22.0-engine/examples/usercmodule/TinyCircuits-Tiny-Game-Engine/filesystem/launcher.py", line 300, in <module>
  File "main.py", line 40, in <module>
NameError: name 'machine' isn't defined
